<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo-Trading</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Cоздать новую позицию</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="longRadio">
         <property name="text">
          <string>Long</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="shortRadio">
         <property name="text">
          <string>Short</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Bitcoin (BTC)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ethereum (ETH)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solana (SOL)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ripple (XRP)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pepecoin (PEPE)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Плечо</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cумма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sumEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addTradeButton">
       <property name="text">
        <string>Создать трейд</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
